### PR TITLE
Basic install and setup requirements

### PR DESCRIPTION
**Pull request by pfraze**
This outlines the most basic requirement definitions, without going deeply into details. (It's meant to be uncontroversial.) One item I can imagine debating is, whether "friending" and "following" should be two distinct concepts. As they do exist in a defacto nature, I suggest we accept them into the requirements and then have future discussions about whether they should or shouldnt exist at all.

rfr @dominictarr and @mixmix

### CODE REVIEW

File: requirements/familiarization.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Familiarization with the UI
+
+The UI should provide cues and tutorials to train the user on the toolset.
```
Comment (mixmix) [inline requirements/familiarization.md]: broadly I agree.
I think the best way to do this is VERY minimally.

I like the ideas of : 
- only bare minimum to start
- 'just in time' as in you get a helper when you stagger into a more complex area
- un-blocking

these are implimentation focused.. perhaps for later conversation

File: requirements/befriend-other-users.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Befriend Other Users
+
+Users should be able to befriend other users.
+
+"Befriending" means, to receive a user's updates and send them yours in a two-way relationship.
```
Comment (mixmix) [inline requirements/befriend-other-users.md]: I really like this idea. I am more and more interested in making this an analogue of our in person interaction.

File: requirements/follow-other-users.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# Follow Other Users
+
+Users should be able to follow other users.
```
Comment (mixmix) [inline requirements/follow-other-users.md]: I wonder about "Listen"

File: requirements/befriend-other-users.md (hunk 2 above)
Comment (mixmix) [inline requirements/befriend-other-users.md]: A friend had an interesting idea - Trust should be measured not just by what you've asserted, but by your actions + interactions. 

I think you can see my friendships more by what I read and communicate that by 'mutual-follow' ...

File: requirements/install-and-setup.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# Install and Setup
+
+## Install
+
+User must be able to install a platform-appropriate installer.
+Installer should:
+
+ - Install the application to a platform-appropriate location without super-user rights.
```
Comment (mixmix) [inline requirements/install-and-setup.md]: and without triggering "non-app store" type warnings

File: requirements/install-and-setup.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+ - Overwrite old versions.
```
Comment (mixmix) [inline requirements/install-and-setup.md]: ensure that users know they are 'upgrading' and not wiping their account

File: requirements/sync-other-users.md [hunk 6]
```diff
@@ -0,0 +1,6 @@
+# Sync Other Users' Data
+
+Users should be able to syncronize past and current data with other users.
+Befriended and followed users should have priority in syncronization.
```
Comment (mixmix) [inline requirements/sync-other-users.md]: and local (wifi) users

File: requirements/find-users-without-prior-relationship.md [hunk 7]
```diff
@@ -0,0 +1,3 @@
+# Find Users without Prior Relationship
+
+Users should be able to locate a user without having any previous interactions within Patchwork.
```
Comment (mixmix) [inline requirements/find-users-without-prior-relationship.md]: and recognise them (as in finding more than  :  [blank_image] @ye33dasd...)

File: requirements/befriend-other-users.md (hunk 2 above)
Comment (pfraze) [inline requirements/befriend-other-users.md]: We've discussed this! Mentioning each other, and replying to each other, are trust signals for instance.

File: requirements/familiarization.md (hunk 1 above)
Comment (pfraze) [inline requirements/familiarization.md]: yeah we can hash that in the proposals

File: requirements/find-users-without-prior-relationship.md (hunk 7 above)
Comment (pfraze) [inline requirements/find-users-without-prior-relationship.md]: aye

File: requirements/follow-other-users.md (hunk 3 above)
Comment (pfraze) [inline requirements/follow-other-users.md]: Hmm, that's interesting

**Comment by pfraze** [comment]
Going to merge, we can continue to build on this.

**Pull request merged**